pkg/fswatcher/fs_watcher.go

```diff
@@ -34,6 +35,11 @@ func (f *fsnotifyWatcherWrapper) Add(name string) error {
 	return f.fsnotifyWatcher.Add(name)
 }
 
+// Close closes the watcher.
+func (f *fsnotifyWatcherWrapper) Close() error {
```

Comment:
please add a test for this func

Reply:
Ok I'll do it.

Is there any way to check Codecov failure locally? I ran `make test` and as I can remember it didn't report it.

Reply:
Not to my knowledge. You can only compare package level coverage before and after the change with go test -cover.